src/components/UserManagement/__tests__/UserSearchPanel.test.js

```diff
@@ -32,10 +75,10 @@ describe('user search panel', () => {
     });
   });
   describe('Behavior', () => {
-    it('should call onNewUserClick() when the user clicks `create new user` button', () => {
-      userEvent.click(screen.getByRole('button', { name: /create new user/i }));
-      expect(onNewUserClick).toHaveBeenCalled();
-    });
+    // it('should call onNewUserClick() when the user clicks `create new user` button', () => {
+    //   userEvent.click(screen.getByRole('button', { name: /create new user/i }));
+    //   expect(onNewUserClick).toHaveBeenCalled();
+    // });
```

Comment:
```suggestion
    it('should call onNewUserClick() when the user clicks `create new user` button', () => {
      userEvent.click(screen.getByRole('button', { name: /create new user/i }));
      expect(onNewUserClick).toHaveBeenCalled();
    });
```
This test should work if the mock user has the 'postUserProfile' permission.